CMakeLists.txt

```diff
@@ -191,8 +200,8 @@ set(test_sourceFiles
     test/queries/test_get_forward_progress_guarantee.cpp
     test/async_scope/test_dtor.cpp
     test/async_scope/test_spawn.cpp
-    test/async_scope/test_spawn_future.cpp
-    test/async_scope/test_empty.cpp
+    # test/async_scope/test_spawn_future.cpp
```

Comment:
Ditto rather work with `#ifdef` in the tests than removing them completely

Reply:
This test no longer needs to be excluded.